backend/src/config/routes.rs

```diff
@@ -39,6 +40,15 @@ pub fn config(cfg: &mut web::ServiceConfig) {
                         .service(layers::find_by_id)
                         .service(layers::create)
                         .service(layers::delete)
+                        .service(
+                            web::scope("/base/images")
+                                .service(base_layer_image::create)
+                                .service(base_layer_image::update)
+                                .service(base_layer_image::delete),
```

Comment:
The layer_id isn't needed for any of the other endpoints. For those the Uuid of the base layer image can be used directly (or as a field in the dto for create).
So if we want to change something I think it would make more sense to remove the layer_id from find and set it as a query param or at the end of the path instead.